spec/controllers/users_controller_spec.rb

```diff
@@ -101,7 +101,7 @@
     context 'with "page"' do
       context 'when using "paged" paginator' do
         before(:all) do
-          JSONAPI.configuration.default_paginator = :paged
```

Comment:
Need to change the resource paginator instead of the default config because of the way `@request.resource_klass._paginator` works in Pagination. If `@paginator` has not been set on the resource, then it is set to the default config paginator. That's all good for the first set of paginator tests, but when it moves on to the next ones and updates the default paginator setting, `@paginator` for UserResource is still set to what it was previously. 

So, in the course of running the "paged" tests, `UserResource._paginator` gets set to `:paged`, even if you never updated the UserResource directly & changed the default paginator setting. When it gets to the "offset" tests and you update the default paginator to "offset", then you'll get an error because `UserResource._paginator` is still set as `:paged`.

Basically, this change is solving an issue with the tests themselves, not an issue being raised by the code.